### ISSUE TITLE
Application error during route planning

### ISSUE THREAD

**Issue by deevroman** [organicmaps/organicmaps#6646]
**Describe the issue**
Application error during route planning. Probably because the starting point is near the border of the region.

**Steps to reproduce**
1.  Set the start of the route to `59.918799, 30.50255`
2. Set the end of the route to the left of the railway line
3. The app refuses to build a route

**Expected behaviour**
A route must be created

**Screenshots**

https://github.com/organicmaps/organicmaps/assets/25708359/7dd89b3d-5f22-4178-98ca-4dd6e9c8b944

**System information:**
 - Android 13 OneUI 5.1.  
 - Organic Maps version: 2013.11.17-17-Web (map data 2023-11-13)
 - Device Model: Samsumg A52

**Comment by biodranik** [comment]
Does the route work in OpenStreetMap.org ?

**Comment by deevroman** [comment]
> Does the route work in OpenStreetMap.org ?

Yes. Moreover, the video at the end shows that if you choose a farther starting point, OrganicMaps will build the route as well.

**Comment by patepelo** [comment]
Strange. Present OSM data seems alright. Ways were edited recently but I haven't found any evidence of past data inaccuracy.

**Comment by vng** [comment]
https://github.com/organicmaps/organicmaps/issues/3815

**Comment by vng** [comment]
Fixed in map data as of 260404.
